CHANGES.md

```diff
@@ -1,5 +1,222 @@
+  schedule_storage:
+  module: dagster_postgres.schedule_storage
+  class: PostgresScheduleStorage
+  config:
+      postgres_url: { PG_DB_CONN_STRING }
+  ```
+
+  You can now write:
+
+  ```
+  storage:
+    postgres:
+      postgres_url: { PG_DB_CONN_STRING }
+  ```
+
+* All assets where a `group_name` is not provided are now part of a group called `default`.
+* The group_name parameter value for `@asset` is now restricted to only allow letters, numbers and underscore.
+* You can now [set policies to automatically retry Job runs](https://docs.dagster.io/master/deployment/run-retries). This is analogous to op-level retries, except at the job level. By default the retries pick up from failure, meaning only failed ops and their dependents are executed.
+* [dagit] The new repository-grouped left navigation is fully launched, and is no longer behind a feature flag.
+* [dagit] The left navigation can now be collapsed even when the viewport window is wide. Previously, the navigation was collapsible only for small viewports, but kept in a fixed, visible state for wide viewports. This visible/collapsed state for wide viewports is now tracked in localStorage, so your preference will persist across sessions.
+* [dagit] Queued runs can now be terminated from the Run page.
+* [dagit] The log filter on a Run page now shows counts for each filter type, and the filters have higher contrast and a switch to indicate when they are on or off.
+* [dagit] The partitions and backfill pages have been redesigned to focus on easily viewing the last run state by partition.  These redesigned pages were previously gated behind a feature flag — they are now loaded by default.
+* [dagster-k8s] Overriding labels in the K8sRunLauncher will now apply to both the Kubernetes job and the Kubernetes pod created for each run, instead of just the Kubernetes pod.
+
+### Bugfixes
+
+* [dagster-dbt] In some cases, if Dagster attempted to rematerialize a dbt asset, but dbt failed to start execution, asset materialization events would still be emitted. This has been fixed.
+* [dagit] On the Instance Overview page, the popover showing details of overlapping batches of runs is now scrollable.
+* [dagit] When viewing Instance Overview, reloading a repository via controls in the left navigation could lead to an error that would crash the page due to a bug in client-side cache state. This has been fixed.
+* [dagit] When scrolling through a list of runs, scrolling would sometimes get stuck on certain tags, specifically those with content overflowing the width of the tag. This has been fixed.
+* [dagit] While viewing a job page, the left navigation item corresponding to that job will be highlighted, and the navigation pane will scroll to bring it into view.
+* [dagit] Fixed a bug where the “Scaffold config” button was always enabled.
+
+### Community Contributions
+
+* Provide dagster-mlflow configuration parameters as environment variables, thanks @chasleslr!
```

Comment:
nitpick - this one doesn't quite parse as a sentence. Add a "You can now" at the front?

Reply:
fixed, thanks!